Task3/EEL_task_3/src/main/java/com/epam/esm/config/SpringConfig.java

```diff
@@ -0,0 +1,89 @@
+package com.epam.esm.config;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.core.env.Environment;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.datasource.DriverManagerDataSource;
+import org.springframework.web.servlet.config.annotation.EnableWebMvc;
+import org.springframework.web.servlet.config.annotation.ResourceHandlerRegistry;
+import org.springframework.web.servlet.config.annotation.ViewResolverRegistry;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
+import org.thymeleaf.spring5.SpringTemplateEngine;
+import org.thymeleaf.spring5.templateresolver.SpringResourceTemplateResolver;
+import org.thymeleaf.spring5.view.ThymeleafViewResolver;
+import springfox.documentation.swagger2.annotations.EnableSwagger2;
+
+import javax.sql.DataSource;
+
+
+@Configuration
+@ComponentScan("com.epam.esm")
+@EnableWebMvc
+@EnableSwagger2
+
+@PropertySource("classpath:database.properties")
+public class SpringConfig implements WebMvcConfigurer {
+    private final ApplicationContext applicationContext;
+    private final Environment environment;
+
+    @Autowired
+    public SpringConfig(Environment environment, ApplicationContext applicationContext) {
+        this.environment = environment;
+        this.applicationContext = applicationContext;
+    }
+
+    @Bean
+    public SpringResourceTemplateResolver templateResolver() {
+        SpringResourceTemplateResolver templateResolver = new SpringResourceTemplateResolver();
+        templateResolver.setApplicationContext(applicationContext);
+        templateResolver.setPrefix("/WEB-INF/views/");
+        templateResolver.setSuffix(".html");
+        return templateResolver;
+    }
+
+    @Bean
+    public SpringTemplateEngine templateEngine() {
+        SpringTemplateEngine templateEngine = new SpringTemplateEngine();
+        templateEngine.setTemplateResolver(templateResolver());
+        templateEngine.setEnableSpringELCompiler(true);
+        return templateEngine;
+    }
+
+    @Override
+    public void addResourceHandlers(ResourceHandlerRegistry registry) {
+        registry.addResourceHandler("/swagger-ui.html**")
+                .addResourceLocations("classpath:/META-INF/resources/swagger-ui.html");
+        registry.
```

Comment:
Why do we need resource handlers?

Reply:
We have annotation  @EnableWebMvc, so we need to add addResourceHandlers to resolve static resources, which are created by Swagger